src/Models/Feature.php

```diff
@@ -23,9 +24,14 @@ public function featured(): MorphTo
 
     public function scopeForBucket(Builder $query, string $bucketKey): Builder
     {
-        return $query->where('bucket_key', $bucketKey)->get()->map(function ($feature) {
+        return $query->where('bucket_key', $bucketKey)->with('featured');
+    }
+
+    public static function getForBucket(string $bucketKey): Collection
+    {
+        return self::forBucket($bucketKey)->get()->map(function (Feature $feature) {
             return $feature->featured;
-        })->filter();
```

Comment:
We may need to keep this filter call to remove empty rows due to deleted records.

Reply:
👍 added it back